src/lib/dataEntities/networks.ts

```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
      */
     depositTimeout: 1800000,
   },
+  421416: {
```

Comment:
Should be 42141**4** no?

https://sepolia.etherscan.io/address/0x13C75Ec6EEBd45837C53D3A91561D88C296fC5A5#readProxyContract (under ChainId)

Reply:
lol you also got it wrong its 421614 not 421416 or 421414